tests/phpunit/tests/admin/includesScreen.php

```diff
@@ -606,6 +637,9 @@ public function data_is_block_editor() {
 	 *     $function string Function name to hook to the filter.
 	 * }
 	 * @param bool   $expected The expected `is_block_editor` value.
+	 *
+	 * @covers ::set_current_screen
+	 * @covers ::get_current_screen
```

Comment:
```suggestion
	 * @covers ::set_current_screen
```

The work to setup the block editor properties is done when running `set_current_screen` as it invokes `get` static method which handles the setup. There is a method called `is_block_editor()` which seems more like the test subject of this specific test. It's currently not though. So without using that method, this test is validating the initial instantiation is done correctly when the current screen is for the block editor.